lib/pbench/server/indexer.py

```diff
@@ -847,15 +847,23 @@ def _known_user_benchmark(ptb):
         try:
             name, ver = first_line.split(" ", 1)
         except ValueError:
-            name = first_line
+            name = first_line.strip()
             ver = ""
-        name = name.strip()
+        else:
+            name = name.strip()
+            ver = ver.strip()
         try:
             ubm_driver_versions = ResultData.known_user_benchmarks[name]
         except KeyError:
             pass
         else:
-            ver = ver.strip()
+            # For compatibiilty with "Hammerdb-tpcc", we'll map an empty
+            # string and "Hammerdb-3.2" to the expected value of
+            # "HammerDB-3.2".
+            if name == "Hammerdb-tpcc" and (not ver or ver == "Hammerdb-3.2"):
+                ver = "HammerDB-3.2"
+            else:
+                ver = ver.strip()
```

Comment:
Why are you re-stripping `ver` here? You already did it at 854 unless you set it to `""` at 851??

Reply:
Fixed.